fluent-plugin-kinesis.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.required_ruby_version = '>= 2.1'
 
-  spec.add_dependency "fluentd", ">= 0.14.10", "< 2"
+  spec.add_dependency "fluentd", ">= 0.14.22", "< 2"
```

Comment:
We have to update dependencies in README `Fluentd 0.14.10+` too.

Reply:
Fixed in https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/174/commits/8d2ac48219536488d69150c3c4f704dd337b6884.